backend/src/main/resources/sql/data-dev.sql

```diff
@@ -16,3 +17,19 @@ VALUES ('민초', 1),
        ('초콜릿 맛 개구리', 4),
        ('언제 죽을 지 알기', 5),
        ('어떻게 죽을 지 알기', 5);
+
+
+INSERT INTO room(total_round, current_round)
+VALUES (5, 1);
+
+
+INSERT INTO room_content(room_id, balance_content_id, round, created_at)
+VALUES (1, 1, 1, '2024-07-25 13:24:09'),
+       (1, 2, 1, '2024-07-25 13:24:09'),
+       (1, 3, 1, '2024-07-25 13:24:09'),
+       (1, 4, 1, '2024-07-25 13:24:09'),
+       (1, 5, 1, '2024-07-25 13:24:09');
```

Comment:
의견) 초기 데이터 수정 부탁드려요~

```suggestion
INSERT INTO room_content(room_id, balance_content_id, round, created_at)
VALUES (1, 1, 1, '2024-07-25 13:24:09'),
       (1, 2, 2, '2024-07-25 13:24:09'),
       (1, 3, 3, '2024-07-25 13:24:09'),
       (1, 4, 4, '2024-07-25 13:24:09'),
       (1, 5, 5, '2024-07-25 13:24:09');
```